src/BasicLogger.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace ReadableUnitTests;
+
+class BasicLogger extends \Sil\Psr3Adapters\LoggerBase
+{
+    /**
+     * Log a message.
+     *
+     * @param mixed $level
+     * @param string $message
+     * @param array $context
+     * @return null
+     */
+    public function log($level, $message, array $context = [])
+    {
+        echo $this->interpolate($message, $context) . PHP_EOL;
```

Comment:
was simply using the `LoggerBase.interpolate` not an option?  Just wondering why you needed this wrapper?

Reply:
LoggerBase was still an abstract class, basically to provide the interpolate code so that all a subclass/implementation would have to do was provide a `log()` method. Thanks for asking, though. :-)